internal/pkg/cli/dialog/use_template_test.go

```diff
@@ -57,7 +57,7 @@ func TestAskUseTemplateOptionsIfMet(t *testing.T) {
 		assert.NoError(t, err)
 
 		time.Sleep(20 * time.Millisecond)
-		_, err = console.SendLine(strings.Repeat(Backspace, 4)) // remove "text"
+		_, err = console.Send(strings.Repeat(Backspace, 4)) // remove "text"
```

Comment:
Sem/tam sa zasekne tento test:
![image](https://user-images.githubusercontent.com/19371734/148925563-07afd8df-1615-4bec-9f38-5c270e1fe45e.png)

Je to preto, ze sa tam chybne posielal aj `\n`, a teda otazka sa zobrazila znova.
No a ak sa poslala nasledujuca odpoved pocas vypisovania otazky 2x, tak sa odpoved nezaznamenala.